index.html

```diff
@@ -1,18 +1,30 @@
 <!DOCTYPE html>
 <html lang="pl">
-
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    <title>TITLE</title>
+  <head>
+    <meta charset="utf-8" />
```

Comment:
Wydaje mi się że możemy to  rozszerzyć 
meta http-equiv="Content-Type" content="text/html; charset=utf-8"